<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PresetEditorWidget</class>
 <widget class="QWidget" name="PresetEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="additionsFrame">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,2,0,1">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="notesLabel">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Notes: </string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QLabel" name="notesValueLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QLabel" name="delayLabel">
        <property name="text">
         <string>Delay  :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="delaySpinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="notationsFrame"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
